Curtains/Curtains.Application/DTO/OrderDTO.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Curtains.Application.DTO
+{
+    public class OrderDTO
+    {
+        #region PropertiesRegion
+        /// <summary>
+        ///  This property represents comment to order.
+        /// </summary>
+        public string Comment { get; set; }
+        public string PhoneNumber { get; set; }
```

Comment:
Why did you leave PhoneNumber and MarketingInfosId properties without documentation?